ground-core/docker/config.yml

```diff
@@ -0,0 +1,17 @@
+dbType: neo4j
```

Comment:
Got it. We might eventually want to write a `build_docker.sh` that copies the config file and starts the container or something, but don't worry about it for now.